micah-akpan/questioner-api
# #163229253 Enable User to Login using Username

#### What does this PR do?
 It adds support for login to the app using username
 The previous implementation only used email
#### Description of Task to be completed?
`/auth/login/` supports login with email or username
#### How should this be manually tested?
- clone repo
- `npm install`
- `npm run dev:start`
- Launch Postman, create an account with the required fields
   - firstname: [string],
   - email: [string],
   - lastname: [string]
- Add an authorization token with the obtained token
- Go to the /auth/update to update user details and add a username (this endpoint will be built soon)
- Sign In using the username
- You should be able to sign in by passing your username and password in the request
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#163229253
#### Screenshots (if appropriate)
#### Questions:
